src/bp/core/events/event-collector.ts

```diff
@@ -1,15 +1,14 @@
 import * as sdk from 'botpress/sdk'
 import { ConfigProvider } from 'core/config/config-loader'
 import Database from 'core/database'
-import { EventRepository } from 'core/repositories'
+import { SessionIdFactory } from 'core/services/dialog/session/id-factory'
 import { TYPES } from 'core/types'
 import { inject, injectable, tagged } from 'inversify'
 import Knex from 'knex'
 import _ from 'lodash'
 import moment from 'moment'
 import ms from 'ms'
-
-import { SessionIdFactory } from '../dialog/session/id-factory'
+import { EventRepository } from './event-repository'
```

Comment:
I'm not sure yet, I like the idea of using "absolute" paths for files outside your current folder (to avoid ../../..), but I kind of like the idea of having related files / child files easily identifiable (with ./ )